standards/README.md

```diff
@@ -15,3 +15,4 @@ This folder contains all current standards for software development.
 - [Security standards](security_standards.md)
 - [TSQL and SQL Server database standards](tsql_and_sqldb_standards.md)
 - [Version control standards](version_control_standards.md)
+- [Docker standards](docker_standards.md)
```

Comment:
As it stands, a corresponding entry _also_ needs adding in the main README at the root of the repo.  We may want to look at that though...

Reply:
Added